packages/d/docsearch-js.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "docsearch-js",
+  "filename": "umd/index.js",
+  "description": "JavaScript package for DocSearch, the best search experience for docs.",
+  "keywords": [
+    "doc",
+    "search"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/algolia/docsearch.git"
+  },
+  "homepage": "https://docsearch.algolia.com/",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@docsearch/js",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
:wave: We try to avoid using open globstars in configs as it can lead to including files unintentionally -- looking at the package, you can likely use `@(esm|umd)/*.@(js|d.ts)`

Reply:
ok